src/HomeScreen.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react";
+
```

Comment:
Add interface here to define what type the props will be

Reply:
still need to learn typescript :( 😢